packages/x-charts/src/internals/plugins/featurePlugins/useChartPolarAxis/useChartPolarAxis.ts

```diff
@@ -71,24 +76,21 @@ export const useChartPolarAxis: ChartPlugin<UseChartPolarAxisSignature<any>> = (
   }, [seriesConfig, drawingArea, rotationAxis, radiusAxis, dataset, store]);
 
   const svg2rotation = React.useCallback(
-    (x: number, y: number) => Math.atan2(x - center.cx, center.cy - y),
+    (x: number, y: number) => generateSvg2rotation({ cx: center.cx, cy: center.cy })(x, y),
```

Comment:
```suggestion
    generateSvg2rotation({ cx: center.cx, cy: center.cy }),
```

Reply:
Unfortunatly, the react linter will complain it does not know the function and so can not verify the dependencies of the callback.

Reply:
If you return a function, our can use `useMemo` instead of `useCallback`.
Btw `svg2rotation` is a weird function name :eyes: 

```ts
const getSVGFromRotation = React.useMemo(() => generateSvg2rotation({ cx: center.cx, cy: center.cy }, [center]);

// Or maybe even simpler, I did not check if that would cause problems with other center properties:
const getSVGFromRotation = React.useMemo(() => generateSvg2rotation(center), [center]);
```

Reply:
It translates svg coordinates to the rotation coordinate. xxx2yyy to do a mapping between two coordinate space seems quite frequent

Reply:
If it's clear to the chart team, it's the most important :pray: 
I'm personally not a fan of function whose name don't start with a verb, but it's clearly subjective.